draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -251,6 +251,23 @@ same address and either the same or a different port; this includes cases
 where the signaled candidate is discarded as redundant according to 
 Section 5.1.3 of {{RFC8445}}.
 
+Interactions with TURN servers 
+------------------------------
+
+When sending data to a TURN {{RFC5766}} server, the sending client tells
```

Comment:
Might need to add a reference to RC5766 around line 40 of the document.

Reply:
I think I did this...